src/mame/atari/batman.cpp

```diff
@@ -452,7 +451,7 @@ void batman_state::batman(machine_config &config)
 
 	TILEMAP(config, "vad:playfield", "gfxdecode", 2, 8, 8, TILEMAP_SCAN_COLS, 64, 64).set_info_callback(FUNC(batman_state::get_playfield_tile_info));
 	TILEMAP(config, "vad:playfield2", "gfxdecode", 2, 8, 8, TILEMAP_SCAN_COLS, 64, 64, 0).set_info_callback(FUNC(batman_state::get_playfield2_tile_info));
-	TILEMAP(config, "vad:alpha", "gfxdecode", 2, 8, 8, TILEMAP_SCAN_ROWS, 64, 32, 0).set_info_callback(FUNC(batman_state::get_alpha_tile_info));
+	TILEMAP(config, "vad:alpha", "gfxdecode", 2, 8, 8, TILEMAP_SCAN_ROWS, 64, 30, 0).set_info_callback(FUNC(batman_state::get_alpha_tile_info));
```

Comment:
Why the reduction in size here?  How much tile map memory is there backing the alpha tilemap?

Reply:
In this case, alpha tilemap memory size is less than 64 * 32 words, And it results of potentially crash and glitch at bottom most area. That's why I reduced tilemap size.